src/header/name.rs

```diff
@@ -1759,32 +1759,23 @@ impl From<HeaderName> for Bytes {
     }
 }
 
-impl<'a> HttpTryFrom<&'a HeaderName> for HeaderName {
```

Comment:
Aren't the impls of `TryFrom<&'_ Self>` still needed, so as to clone itself?

Reply:
Rustc complains:

```
error[E0119]: conflicting implementations of trait `std::convert::TryFrom<&header::name::HeaderName>` for type `header::name::HeaderName`:
    --> src/header/name.rs:1778:1
     |
1778 | impl<'a> TryFrom<&'a HeaderName> for HeaderName {
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: conflicting implementation in crate `core`:
             - impl<T, U> std::convert::TryFrom<U> for T
               where U: std::convert::Into<T>;
```